### ISSUE TITLE
Rule: Disallow multiple spaces [$15 awarded]

### ISSUE THREAD

**Issue by nzakas** [eslint/eslint#630]
This rule should warn when there are multiple consecutive spaces inside of an expression, such as:

``` js
if (foo   === "bar") {}
```

The rule ID should be `no-multi-spaces` and be on by default.
##
